src/api.rs

```diff
@@ -97,15 +97,11 @@ impl QuicP2p {
 
         let mut bootstrap_cache =
             BootstrapCache::new(cfg.hard_coded_contacts, custom_dirs.as_ref())?;
-        if use_bootstrap_cache {
```

Comment:
I think it'll be useful o have some log entry stating that bootstrap cache is being used and which are the nodes being used from the cache.